style.css

```diff
@@ -42,12 +42,20 @@ p {
 }
 
 #content aside {
+  backdrop-filter: blur(2px);
```

Comment:
I stumbled across this CSS property while researching ways to blur the praise on load. It ended up not doing what I'd hoped -- applied at this element, it blurs the background behind the praise, but not the text itself.

But I think it adds a nice effect so I kept it! 

<img width="244" alt="image" src="https://github.com/pepopowitz/doing-a-good-job/assets/1627089/9d9218dd-7129-41e0-b2cd-8be9c4d10a25">